### DISCUSSION TITLE
How to make a Status page on website?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [upptime/upptime#740] [Q&A] [answered]
How can I make the status page on my website?

**Comment by AnandChowdhary** [comment, marked as answer]
It's a good question! :) We offer a get started guide here: https://upptime.js.org/docs/get-started

**Comment by KendallDoesCoding** [comment]
Can you help me set it up, please?

**Reply by AnandChowdhary** [reply]
Happy to help -- is there any particular step of the "Getting started" guide that you're having trouble with?
